### DISCUSSION TITLE
V204 of SavvyCAN

### DISCUSSION THREAD

**Discussion by collin80** [collin80/SavvyCAN#335] [General]
In the past week or two a lot has happened. I released V203 then V204. V204 is probably a pretty big leap forward if you use DBC files. Now signals are used in the Frame Data Analysis window and File Comparator. Also, graphing of multiplexed signals is now properly possible. And, the graphing window will allow itself to be opened multiple times so you can have separate graphs going all at once. Additionally, graphs of signals with value tables will now show the value strings on the graph so you can better figure out what is going on.

And, one thing I failed to mention in GIT commits or on the website is that you can now right click on signals on the main screen when you're looking at traffic. This allows you to directly graph that signal either in the last opened window or a new window.

One thing is for sure, I've been abusing the continuous builds a bit too much. They are WAY WAY ahead of the latest "release" now. It's probably like a night and day difference in how everything works. I'll try my best to get some videos of all this new stuff to help show how to use it. And, I'll try to update the help documentation at some point too.

**Comment by Breastfed** [comment]
Hi Collin
FYI V204 The dbc editor randomly crash's after  adding a new message or it wont save the changes for the Frame ID. I found that if I exit the editor and save the changes in the manager then I could re-open my saved file in the editor and then add a new Signal... this was a more successful approach however this often crashed as well or I ended up with a completely corrupted the dbc file ( 0bytes) 

BTW V202 I'm making progress and seems to work fine.

Breastfed

**Comment by DrewGauge** [comment]
How do we download V204?

Also on 17+ gm ecu I cant view lowspeed any ideas? Looking for steering wheel control pids

**Reply by collin80** [reply]
The best download at this point is the development / continuous build: https://github.com/collin80/SavvyCAN/releases/tag/continuous

It is currently showing V204. I haven't released a proper versioned build in some time - really since I started doing continuous builds that fire every time a commit is done. So, the best approach is to grab that build as it has all the most up to date code and fixes.
